drake/util/drakeUtil.h

```diff
@@ -24,7 +24,11 @@
     #define DLLEXPORT
 #endif
 
-
+template <typename Derived>
+inline void sizecheck(const Eigen::MatrixBase<Derived>& mat, unsigned int rows, unsigned int cols){
```

Comment:
these unsigned ints everywhere are making MSVC complain (rightly so). I think size_t is the right type in all cases where unsigned int is currently used.